kernel/Dockerfile.kconfig

```diff
@@ -17,7 +17,7 @@ RUN set -e && for VERSION in ${KERNEL_VERSIONS}; do \
         echo "Downloading/Unpacking $VERSION" && \
         KERNEL_SOURCE=https://www.kernel.org/pub/linux/kernel/v4.x/linux-${VERSION}.tar.xz && \
         [ -f sources/linux-${VERSION}.tar.xz ] || curl -fSLo sources/linux-${VERSION}.tar.xz ${KERNEL_SOURCE} && \
-        tar xf sources/linux-${VERSION}.tar.xz; \
+        bsdtar xf sources/linux-${VERSION}.tar.xz; \
```

Comment:
this should work with GNU tar with `-xjf`

Reply:
It doesn't (with `-xJf`). I get the same error. I think this may be an issue with `overlay2` as google comes up with a bunch of hits with the same error